src/auth-stack.ts

```diff
@@ -396,9 +401,25 @@ export class AuthStack {
     const graphQLHandler = stack.node.tryFindChild('graphql-handler') as Function
     graphQLHandler?.addEnvironment(JWT_ENV_VARS.BOOSTER_JWKS_URI, jwksUri)
     graphQLHandler?.addEnvironment(JWT_ENV_VARS.BOOSTER_JWT_ISSUER, issuer)
+    graphQLHandler?.addEnvironment(JWT_ENV_VARS.BOOSTER_ROLES_CLAIM, 'cognito:groups')
 
     const subscriptionHandler = stack.node.tryFindChild('subscriptions-notifier') as Function
     subscriptionHandler?.addEnvironment(JWT_ENV_VARS.BOOSTER_JWKS_URI, jwksUri)
     subscriptionHandler?.addEnvironment(JWT_ENV_VARS.BOOSTER_JWT_ISSUER, issuer)
+    subscriptionHandler?.addEnvironment(JWT_ENV_VARS.BOOSTER_ROLES_CLAIM, 'cognito:groups')
+  }
+
+  /**
+   * It will create all the groups based on roles definitions
+   * @param resourceParams current resource params
+   */
+  private static createGroups(resourceParams: ResourceParams): void {
+    const { stack, config, rocketStackPrefixId, userPool } = resourceParams
+    Object.entries(config.roles).map((role: any) => {
```

Comment:
Could we define the role type instead of using any here?

Reply:
Agree, I'd use here `role: Array<string>`. (I believe `groupName` is a string right?)

Reply:
Each role is a `[string, RoleMetadata]`. Fixed!